### ISSUE TITLE
Validation doesn't work!

### ISSUE THREAD

**Issue by mohammedzamakhan** [formly-js/ng-formly-nativescript#6]
I tried adding validation, by adding validators and validationMessages for required on FormlyModule.forRoot, and then I had required: true to a field, but it didn't seem working

**Comment by NathanWalker** [comment]
@mohammedzamakhan Yep at this point it's a proof of concept. Thanks for tracking this, definitely a TODO.

Btw, the structure of the code right now includes the same code as the initial library, however much can be shared, therefore we will probably remove a lot of that code from here and only provide the modified services necessary to provide the NativeScript specific functionality.

For instance, [these files](https://github.com/formly-js/ng-formly-nativescript/tree/master/src/app/core/templates) can all be used *as-is* straight from the original lib; they can just be imported from the installed [orig lib](https://github.com/formly-js/ng2-formly/tree/master/src/core/templates). They are duped here at the moment just to develop things out, but a final step would be to remove all code that is untouched and use those directly from the orig lib. Just stating for other contributors understanding.

I wanted to kick off this development with this POC. Holidays are busy time for me so don't have a timeline to completion at moment but look forward to when that happens; it's a great lib and having it for {N} apps just makes it that much sweeter 👍

**Comment by mohammedzamakhan** [comment]
I am loving NativeScript platform too now. When I created to work on Angular 2 support, I knew that this could be supported for NativeScript, but did not have time to learn it and develop it. You have been a great help for it, and the POC looks clean and smooth.

Let me know if you need any help, will be always open to any new suggestions for the whole library.
